Source/WebCore/svg/properties/SVGList.h

```diff
@@ -194,9 +194,9 @@ class SVGList : public SVGProperty {
 
     virtual void detachItems() { }
     virtual ItemType at(unsigned index) const = 0;
-    virtual ItemType insert(unsigned index, ItemType&&) = 0;
-    virtual ItemType replace(unsigned index, ItemType&&) = 0;
-    virtual ItemType remove(unsigned index) = 0;
+    virtual ItemType insertAt(unsigned index, ItemType&&) = 0;
+    virtual ItemType replaceAt(unsigned index, ItemType&&) = 0;
+    virtual ItemType removeAt(unsigned index) = 0;
```

Comment:
Please mention these in the commit message as well so it's clear they are intentional.